examples/layout/three-column/sunset-delorean-right/index.html

```diff
@@ -11,45 +11,40 @@
 
         <!-- the main content //-->
         <div class="column--first">
-          <div class="collapse">
-            <div class="column--first-header">
-              <h2>Page title that spans two columns</h2>
-            </div>
+          <h2>Page title that spans two columns</h2>
 
+          <div class="nested-grid">
```

Comment:
@kgcreative, thanks for this example. This does exactly what I want it too but it still has one extra div just for the collapse. This is a better place for it than where I was placing it.